x-pack/filebeat/module/crowdstrike/falcon_audit/manifest.yml

```diff
@@ -0,0 +1,8 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/crowdstrike/falconhoseclient/output
```

Comment:
I just realized that both of these filesets read from the same file. Filebeat won't allow two concurrent readers of the same file because it persists the read offset of the file just once in its registry. 

So would you ever expect both filesets enabled the same machine? If the goal of splitting them is to minimize processing of certain events that we don't care about then perhaps a good solution is to have a single fileset with a config option to control what event types it consumes.

You could pass a parameter into your module and then into your script.

```
processors:
- script:
    lang: javascript
    params:
      mode: [audit] # or [audit, endpoint]
```

Reply:
We should make it one fileset/pipeline then. It will always be the same output file. I was already a bit nervous about separating it like this given the additional overhead of doubling the JSON decoding/parsing. The purpose as suggested by @webmat was to semantically separate the two types of event data. Let me see if I can set the dataset in the Javascript pipeline and accomplish the same result there.